store/src/main.rs

```diff
@@ -17,7 +30,114 @@ async fn main() -> Result<()> {
         Command::Serve { .. } => {
             server::serve(config.store, db).await?;
         },
+        Command::Query(command) => query(config, command).await?,
     }
 
     Ok(())
 }
+
+async fn query(
+    config: StoreTopLevelConfig,
+    command: Query,
+) -> Result<()> {
```

Comment:
Let's add some basic comments to this function to explain what it does at the high level (e.g., that it is not going directly to the database but rather is making a request to the stope's gRPC API).